joinfaces-starters/security-spring-boot-starter/build.gradle

```diff
@@ -0,0 +1,13 @@
+description = 'Security Spring Boot Starter'
+jar.manifest.attributes('Automatic-Module-Name': 'joinfaces.starter.security')
+
+dependencies {
+    compile project(':joinfaces-starters:jsf-spring-boot-starter')
+    compile project(':spring-security-jsf-taglib')
```

Comment:
Should we rename the taglib to `joinfaces-security-taglib` to better match the other artifact names?

Reply:
yes.

Reply:
I've renamed spring-security-jsf-taglib to joifnaces-security-taglib.